game/scripts/npc/items/item_bfury.txt

```diff
@@ -53,7 +53,7 @@
       "01"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "bonus_damage"                                    "45 60 85 125 180"
+        "bonus_damage"                                    "45 60 120 180 240"
```

Comment:
Change this to: 45 65 95 135 185

Reply:
Can I get an explanation of why this item has to give abysmal damage. Im assuming the item was nerfed before fools gold was a thing and when quell was percentage based. Even with my buffs to 240 damage this item still gives lower DPS then all the other damage items (Bar nullifier but the active is far superior). The main strength of this item, which is allowing you to flash farm is irrellevant after 10 minutes (why would you upgrade it past level 2). If you need it to clear summons, radiance + octarine + shivas on anyone on your team is far superior, and will make them unkillable.

Reply:
Battlefury is supposed to be **situational**, thats why I said buff cleave percent instead. I proposed those numbers because I used the same scaling logic on other dmg items: x/x+20/x+50/x+90/x+140, so mostly for consistency reasons. If you think damage is still crap DPS-wise, increase the cleave percent even more. If that is not enough, buff regen stats as well.

> If you need it to clear summons, radiance + octarine + shivas on anyone on your team is far superior, and will make them unkillable. 

Lycan wolves, Brood spiders and Enigma Eidolons have more magic resistance than armor. So cleave would be more effective against them.

Reply:
Battle fury will still be situational after these buffs, you wont be buying a battlefury every game. BF is an item like echo sabre or armlet. These items need a little extra stat wise because they arent designed to be good late game items.